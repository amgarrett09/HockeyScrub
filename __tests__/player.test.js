/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import Player from '../pages/player';

describe('Player page', () => {
  it('should render and should match snapshot', () => {
    const props = {
      id: '123456',
      name: 'Jane Doe',
      currentTeam: 'Detroit Red Wings',
      position: 'Goalie',
      shoostCatches: 'L',
      height: '5\' 6"',
      weight: 140,
      nationality: 'USA',
      active: true,
      seasonStats: [],
      careerSeason: {},
      playoffStats: [],
      careerPlayoffs: {},
      lastFive: [],
    };

    const page = renderer.create(<Player {...props} />);
    const json = page.toJSON();
    expect(json).toMatchSnapshot();
  });
});
