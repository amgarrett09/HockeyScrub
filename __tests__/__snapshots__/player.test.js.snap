// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player page should render and should match snapshot 1`] = `
<main
  className="main--visible"
>
  <section
    className="section"
  >
    <h1
      className="title is-1"
    >
      Jane Doe
    </h1>
    <p>
      <strong>
        Detroit Red Wings
      </strong>
    </p>
    <p>
      Position: Goalie • Shoots/Catches: 
    </p>
    <p>
      5' 6" • 140 lbs • USA
    </p>
  </section>
  <section
    className="section"
  >
    <div
      className="container is-fluid"
    />
  </section>
</main>
`;
